backend/app/assets/stylesheets/spree/backend/_bootstrap_custom.scss

```diff
@@ -34,7 +34,7 @@ $gray-lightest:             #f7f7f9 !default;
 
 $brand-primary:             $color-3 !default;
 $brand-success:             $color-2 !default;
-$brand-info:                #5bc0de !default;
+$brand-info:                #eff5fc !default;
```

Comment:
I don't this is an appropriate variable to store the colour for the card backgrounds. This value should be usable for buttons, alerts, labels etc. Like in [this bootstrap example](https://v4-alpha.getbootstrap.com/components/buttons/#examples).

This won't work if we use it as a common background because it will totally blend in. It's also too light for use outside. 

![](http://i.hawth.ca/s/LycF5BSX.png) (`btn-info` on the right)

This should probably be in a dedicated variable.